io.gitlab.Goodvibes.yaml

```diff
@@ -0,0 +1,38 @@
+---
+app-id: io.gitlab.Goodvibes
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: goodvibes
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--socket=pulseaudio'
+  # Until a sandbox-compatible backend is available, applications that require
+  # access to dconf can do so with the following options:
+  - '--filesystem=xdg-run/dconf'
+  - '--filesystem=~/.config/dconf:ro'
+  - '--talk-name=ca.desrt.dconf'
+  - '--env=DCONF_USER_CONFIG_DIR=.config/dconf'
+  # MPRIS2 support
+  - '--own-name=org.mpris.MediaPlayer2.Goodvibes'
+modules:
+  - name: keybinder
+    sources:
+      - type: archive
+        url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+        sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+    cleanup: [ /include, /lib/*.la, /lib/girepository-1.0, /lib/pkgconfig, /share ]
+
+  - name: goodvibes
+    buildsystem: meson
+    builddir: true
+    sources:
+      - type: git
+        url: https://gitlab.com/goodvibes/goodvibes.git
+        tag: v0.4.2
+        commit: 7559cfc0291871fa05acadb4ff3f19a4a89c997c
+cleanup:
```

Comment:
NIT: This usually goes after `finish-args` :innocent:

Reply:
OK, no worries, I moved the `cleanup` field up there after `finish-args` in 5afe9e6. Thanks!